recipes/ordpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ordpy" %}
+{% set version = "1.0.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ordpy-{{ version }}.tar.gz
+  sha256: 45a10bcad068854a4e0f76144d0d395c0fd702ea2ea3dbb469e7501be95ec7ee
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - ordpy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/arthurpessa/ordpy
+  summary: 'A Python package for data analysis with permutation entropy and ordinal networks methods.'
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add a comment in the above the `license_file` line noting that the license file is manually packaged. I opened an issue upstream (https://github.com/arthurpessa/ordpy/issues/1) to track resolving this. Also add the link as a reminder to check in future releases if the manually packaged copy can be removed here once it's in the source distribution.